CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## Version 5.4.0 - 2024-02-03
```

Comment:
It's possible we could get away with it if the classes were not being used but removing properties from a model is a breaking change and should be reflected in the version number

Reply:
@AQI-DanG 

There is no breaking change since we remained the same field orders as before. 

And I am not sure when we should bump up shared library major version. We can either bump up every year/quarter since we have a Jira release version bumps, or we can bump up when we upgrade with dotnet versions. Not sure one is best, but we'd better to have a DL to stand out that policy.

Reply:
The major version should be bumped on a breaking change per semver guidelines. I would argue that removing a property and removing a type would be a breaking change.  Since those models are not in use anywhere but the code you are writing I could possibly be convinced it is not necessary in this case but there is also not a real good reason not to bump the major version that I can think of.

Reply:
@AQI-DanG , the version has been upgraded to 6.0.0 and submodule reference commit is updated as well.